package.json

```diff
@@ -90,7 +90,7 @@
     "@mapeo/schema": "3.0.0-next.13",
     "@react-native-community/cli": "^12.3.6",
     "@react-native/babel-preset": "^0.73.21",
-    "@react-native/eslint-config": "^0.73.2",
+    "@react-native/eslint-config": "^0.75.0-main",
```

Comment:
Not strictly related, but figured I'd update.

Reply:
probably harmless but always a little bit hesitant to update deps that are part of React Native's monorepo, which tend to be tightly tied to specific React Native versions. Wondering if it would make more sense to pin this to a specific version and only update it in tandom with React Native

a template react native app, as as example:

https://github.com/react-native-community/rn-diff-purge/blob/c29cc8066d7249bf0a6be017fb117f991c6be4ba/RnDiffApp/package.json#L21